install/mysql/glpi-empty.sql

```diff
@@ -9935,4 +9936,19 @@ CREATE TABLE `glpi_groups_items` (
   KEY `type` (`type`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci ROW_FORMAT=DYNAMIC;
 
+DROP TABLE IF EXISTS `glpi_assets_customfields`;
+CREATE TABLE `glpi_assets_customfields` (
+  `id` int unsigned NOT NULL AUTO_INCREMENT,
+  `assets_assetdefinitions_id` int unsigned NOT NULL,
+  `name` varchar(255) NOT NULL,
+  `label` varchar(255) NOT NULL,
+  `type` varchar(255) NOT NULL,
+  `field_options` json,
+  `itemtype` VARCHAR(255) NULL DEFAULT NULL,
```

Comment:
IMHO, the `field_options` should only contain the specific options that are not shared by every field type.
For instance, it should contains `min`/`max`/`step` options of the `NumberType` and the `itemtype` option of the `DropdownType` (and the `itemtype` field should then not be present in the table).
In contrary, the common options, like the mandatory and readonly options should probably all have their own field in the table.

Reply:
Field options are anything that can be directly passed as the field_options param of the field macros. Itemtype isn't one of them and "dropdown" isn't really a field type on its own. Rather it is Computer, Ticket, Category, etc.

Reply:
We can keep it like that, but I find it weird to have to get the `multiple` flag from the `$this->custom_field->fields['field_options']['multiple']` and the `itemtype` from `$this->custom_field->fields['itemtype']` (I tried to read it from `$this->custom_field->fields['field_options']['itemtype']` while working on the search options handling).